Sample-ImportFiles/Portfolios.csv

```diff
@@ -2,10 +2,10 @@ PortfolioId,PortfolioName,PortfolioType,PortfolioCurrency,PortfolioCompany,Legal
 105,FundApps AM USA Ltd,Entity,USD,FundApps AM USA,105,105,105,,,15/08/2015,20 Main Street,124348,,,BI598741123,New York,FundApps AM USA Ltd,,"CA-AMRS,UKIM,US-QII",Adam,Smith,US,,adam.smith@fundappsus.com,US,,+1 20 9766 7777,,248958,23957487,65401681,10007,New York,79 Madison Avenue ,New York,FundApps Inc,US,adam.smith@fundappsus.com,0207 154 1544,Adam,Smith,0207 154 1544,Compliance Manager,EC1Y 2BJ,NY ,FUN04321,+1 20 9766 7766,IV,
-12,Tax Haven Fund,Portfolio,EUR,FundApps Alternatives,105,105,,"Disclosure,Validation,Validation Disclosure",12/09/2013,15/08/2015,10 High Street,124348,FALSE,FUN01234,BI598741123,London,FundApps Ltd,15643045,,John,Smith,GB,HSBC ,john.smith@fundappsai.com,GB,,+44 20 9766 7777,,948958,3987487,65401681,EC1Y 2BJ,,10 High Street,London,FundApps,GB,John@fundapps.co,0207 154 1544,John,Smith,0207 154 1544,Compliance Manager,EC1Y 2BJ,,FUN01234,+44 20 9766 7766,IV,564887842
-11,Global Macro Fund,Portfolio,EUR,FundApps Alternatives,103,,103,"Disclosure,Validation,Validation Disclosure",12/08/2013,,10 High Street,,FALSE,,,London,FundApps Alternatives Ltd,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-16,Managed (Voting) Fund,Portfolio,EUR,FundApps Alternatives,103,,103,"Disclosure,Validation,Validation Disclosure",12/01/2013,,,,FALSE,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-15,Managed Fund,Portfolio,GBP,FundApps Alternatives,103,,103,"Disclosure,Validation,Validation Disclosure",12/03/2013,,,,FALSE,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-13,Special Interest Fund,Portfolio,EUR,FundApps Alternatives,13,13,,"Disclosure,Validation,Validation Disclosure",12/06/2013,,,,FALSE,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-14,US Fund I,Portfolio,USD,FundApps AM USA,100,,100,"Disclosure,Validation,Validation Disclosure",12/11/2013,,,,FALSE,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-17,US Fund II,Portfolio,EUR,FundApps AM USA,105,,105,"Disclosure,Validation,Validation Disclosure",12/01/2013,,,,FALSE,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+12,Tax Haven Fund,Portfolio,EUR,FundApps Alternatives,105,105,,"Disclosure, Validation, Validation Disclosure, Short Selling, Short Selling Ban",12/9/2013,15/08/2015,10 High Street,124348,FALSE,FUN01234,BI598741123,London,FundApps Ltd,15643045,,John,Smith,GB,HSBC ,john.smith@fundappsai.com,GB,,+44 20 9766 7777,,948958,3987487,65401681,EC1Y 2BJ,,10 High Street,London,FundApps,GB,John@fundapps.co,0207 154 1544,John,Smith,0207 154 1544,Compliance Manager,EC1Y 2BJ,,FUN01234,+44 20 9766 7766,IV,564887842
+11,Global Macro Fund,Portfolio,EUR,FundApps Alternatives,103,,103,"Disclosure, Validation, Validation Disclosure, Short Selling, Short Selling Ban",12/8/2013,,10 High Street,,FALSE,,,London,FundApps Alternatives Ltd,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
@fabianengler have you opened up the csv file in excel after you made the amendments? I think the date formatting may need to be updated - we've lost the '0' before all the months e.g 12/08/2013 has become 12/8/2013.

Unfortunately excel doesn't just save the previous format by defualt.